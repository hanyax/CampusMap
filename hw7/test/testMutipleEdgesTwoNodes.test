# This test if the graph can have multiple different edge between two points

# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1

# Add edges
AddEdge graph1 n1 n2 0.5
AddEdge graph1 n1 n2 0.1
AddEdge graph1 n1 n2 0.2
AddEdge graph1 n1 n2 0.7

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1
