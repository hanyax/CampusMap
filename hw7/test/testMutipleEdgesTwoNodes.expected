# This test if the graph can have multiple different edge between two points

# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1

# Add edges
added edge 0.500 from n1 to n2 in graph1
added edge 0.100 from n1 to n2 in graph1
added edge 0.200 from n1 to n2 in graph1
added edge 0.700 from n1 to n2 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(0.100) n2(0.200) n2(0.500) n2(0.700)
